### PR TITLE
README.md improved with more examples

### PR DESCRIPTION
**Pull request by seifreed**
**Checklist**

- [X] I wrote some documentation

**Description**

Added more documentation with examples for the YARA generation process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,64 @@ Usage: yrg [action] [args..]   load and run yara rules inside r2
...
+
+    yrg - Initialize a YARA rule.
+    yrgs - Add strings as patterns.
+    yrgx - Add hex patterns.
+    yrgf - Add function byte signatures.
+    yrgz - Add all strings from the current function.
+
+To start using r2yara to create YARA rules automatically, follow these steps:
+
+##### Open a binary with radare2:
```
Comment (trufae) [inline README.md:58]: dont use titles only for enumerating, 5 subtitltes is too much. better just use bold

**Review by trufae — CHANGES_REQUESTED** [review]
^

File: README.md [hunk 2]
```diff
@@ -94,7 +94,9 @@ ys <rule_name>
 ```
 
 ##### To scan the binary with the loaded rules:
-```
+
+missing newline
```
Comment (trufae) [inline README.md]: 🙄

**Comment by sylvainpelissier** [comment]
Hi @seifreed thank you for your contribution, can you squash your commits into a single one please ?

**Comment by trufae** [comment]
@sylvainpelissier you can squash them directly in the web no need to make him do it by hand
Screenshot 2024-10-21 at 16 18 38

**Comment by sylvainpelissier** [comment]
@trufae it seems something is different for me
Screenshot from 2024-10-21 16-26-57

**Pull request merged**